.github/workflows/build.yaml

```diff
@@ -27,6 +27,7 @@ jobs:
           nose2 -v --pretty-assert
   test:
     strategy:
+      fail-fast: false
```

Comment:
Handy! I know it's somewhat OT, but thanks for sharing this. I'll have to use it in other projects.